frontend/src/packages/sites/sites-api.ts

```diff
@@ -50,14 +50,19 @@ class SitesApi extends HttpApi {
     parameters,
     queryParameters,
   }: SitesGetByProjectIdRequestDto): Promise<SiteGetAllResponseDto> {
-    const searchParameters = new URLSearchParams(queryParameters).toString();
+    const searchParameters = new URLSearchParams();
```

Comment:
```suggestion
    const searchParameters = new URLSearchParams();
```
pass parameters as object to URLSearchParams